src/storage/storage.cc

```diff
@@ -236,11 +236,15 @@ Status Storage::CreateColumnFamilies(const rocksdb::Options &options) {
   rocksdb::ColumnFamilyOptions cf_options(options);
   auto res = util::DBOpen(options, config_->db_dir);
   if (res) {
-    std::vector<std::string> cf_names = {kMetadataColumnFamilyName, kZSetScoreColumnFamilyName,
-                                         kPubSubColumnFamilyName,   kPropagateColumnFamilyName,
-                                         kStreamColumnFamilyName,   kSearchColumnFamilyName};
+    std::vector<std::string> cf_names_except_default;
+    const auto &all_column_family = ColumnFamilyConfigs::ListAllColumnFamily();
+    for (const auto &cf : all_column_family) {
+      if (cf.Name() != rocksdb::kDefaultColumnFamilyName) {
+        cf_names_except_default.emplace_back(cf.Name());
+      }
+    }
```

Comment:
all_column_family | transform([](..){ name }) | remove_if([](..){ v != default; })

Reply:
Anyway the logic is just same here...

Reply:
I'm thinking if we can move it into a function, like ColumnFamilyConfigs::ListColumnFamiliesWithoutDefault();